src/addrman.cpp

```diff
@@ -64,36 +64,39 @@ int AddrInfo::GetBucketPosition(const uint256& nKey, bool fNew, int nBucket) con
     return hash1 % ADDRMAN_BUCKET_SIZE;
 }
 
-bool AddrInfo::IsTerrible(int64_t nNow) const
+bool AddrInfo::IsTerrible(NodeSeconds now) const
 {
-    if (nNow - nLastTry <= 60) { // never remove things tried in the last minute
+    if (now - m_last_try <= 1min) { // never remove things tried in the last minute
         return false;
     }
 
-    if (nTime > nNow + 10 * 60) // came in a flying DeLorean
+    if (nTime > now + 10min) { // came in a flying DeLorean
         return true;
+    }
 
-    if (nNow - nTime > ADDRMAN_HORIZON_DAYS * 24 * 60 * 60) { // not seen in recent history
+    if (now - nTime > ADDRMAN_HORIZON) { // not seen in recent history
         return true;
     }
 
-    if (nLastSuccess == 0 && nAttempts >= ADDRMAN_RETRIES) // tried N times and never a success
+    if (TicksSinceEpoch<std::chrono::seconds>(m_last_success) == 0 && nAttempts >= ADDRMAN_RETRIES) { // tried N times and never a success
         return true;
+    }
 
-    if (nNow - nLastSuccess > ADDRMAN_MIN_FAIL_DAYS * 24 * 60 * 60 && nAttempts >= ADDRMAN_MAX_FAILURES) // N successive failures in the last week
+    if (now - m_last_success > ADDRMAN_MIN_FAIL && nAttempts >= ADDRMAN_MAX_FAILURES) { // N successive failures in the last week
         return true;
+    }
 
     return false;
 }
 
-double AddrInfo::GetChance(int64_t nNow) const
+double AddrInfo::GetChance(NodeSeconds now) const
 {
     double fChance = 1.0;
-    int64_t nSinceLastTry = std::max<int64_t>(nNow - nLastTry, 0);
```

Comment:
nit (feel free to ignore): This variable is still being renamed in the scripted diff in the next commit but it no longer exists.